pax/plugins/signal_processing/PulseShapeDiscrimination.py

```diff
@@ -0,0 +1,146 @@
+import numpy
+import itertools
+
+from pax import plugin, dsputils
+
+class F90(plugin.TransformPlugin):
+    def startup(self):
+        self.dt = self.config['sample_duration'] # Some variable from config
+        self.reference_baseline = self.config['digitizer_reference_baseline']
+
+    def transform_event(self, event):
+        found_pulse = set()
+        for s1 in event.s1s():
+            fderivative = s1.sum_waveform[2:] - s1.sum_waveform[:-2]
+            sderivative = s1.sum_waveform[2:] + s1.sum_waveform[:-2] - 2 * s1.sum_waveform[1:-1]
+            try:
+                starttime = [index for index, (slope, curvature) in enumerate(zip(fderivative, sderivative)) if (slope > 0.20 and curvature > 0.20)][0] + 1
+                print(starttime)
+            except IndexError:
+                continue
+            f10_int = 0.0
+            f20_int = 0.0
+            f30_int = 0.0
+       	    f40_int = 0.0
```

Comment:
__[flake8]__
* [E101] indentation contains mixed spaces and tabs
* [W191] indentation contains tabs

<!-- Comment by [SideCI](https://sideci.com) -->